stimela/cargo/cab/ragavi/parameters.json

```diff
@@ -8,112 +8,112 @@
     "msdir": false, 
     "parameters": [
         {
-            "info": "Gain table to plot", 
-            "dtype": "file", 
-            "required": true, 
-            "name": "table", 
-            "io": "output"
-        }, 
+            "info": "Plot only this antenna, or comma-separated list of antennas. Default plots all", 
+            "name": "ant",
+            "dtype": "list:str", 
+            "required": false,
+            "delimiter": ","
```

Comment:
Currently this param-option is not used (https://github.com/SpheMakh/Stimela/blob/master/stimela/cargo/cab/__init__.py#L141) but rather do the following https://github.com/SpheMakh/Stimela/blob/master/stimela/cargo/cab/wsclean/src/run.py#L81